libkbfs/backpressure_disk_limiter.go

```diff
@@ -785,6 +810,18 @@ func defaultGetFreeBytesAndFiles(path string) (int64, int64, error) {
 	return int64(freeBytes), int64(freeFiles), nil
 }
 
+func (bdl *backpressureDiskLimiter) simpleByteTrackerFromType(typ diskLimitTrackerType) (
```

Comment:
@strib this is the last blocker on this PR. Want me to rename it or can I merge?